extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/KubernetesTaskRunnerConfig.java

```diff
@@ -38,6 +38,7 @@
   @JsonProperty
   public boolean debugJobs = false;
 
+  @Deprecated
```

Comment:
can you add a comment about why this config is deprecated and the new alternative?

Reply:
there should be a change in docs too unless this property is not documented at all.